src/index.js

```diff
@@ -0,0 +1,154 @@
+// import 'regenerator-runtime/runtime';
+// import axios from 'axios';
+const state = {
+    temp: 59,
+    city: 'chicago'
+};
+
+changeCityWeather = () => {
```

Comment:
whoops forgot your `const` prefix there

```suggestion
const changeCityWeather = () => {
```